src/server/models/db/migrations/20230717140537_delete_unused_audits/migration.sql

```diff
@@ -0,0 +1,3 @@
+-- Delete all lstItem audit events.
+DELETE FROM "Audit"
+WHERE type = 'listItem';
```

Comment:
This will delete our "DELETED" records too which need to be on the audit record. It will be safer to just delete based on the AuditEvents that we're removing. 

Take a look at the `any` operator in psql, lmk if any issues

Reply:
Hmm, wouldn't using `any` be a bit much? All we have to do is delete the listItem REMINDER audits right?

Reply:
No, did you check what `any(..)` does? it allows you to match entries based on an array. "is this value any of these enums"

deleting `REMINDER` is too little - but deleting `where type = 'listItem'` is deleting too many records.

See the autogenerated comment in the other migration

```
/*
  Warnings:
  - The values [OUT_WITH_PROVIDER,EDITED,REVIEWED,PUBLISHED,PINNED,UNPINNED] on the enum `AuditEvent` are removed here. If these variants are still used in the database, this will fail.
*/
```

Reply:
Ah I see what you mean now, I blame Google 😅

The [first link from the search results](https://www.postgresqltutorial.com/postgresql-tutorial/postgresql-any/) gave this explanation `The PostgreSQL ANY operator compares a value to a set of values returned by a subquery` and none of the examples I saw used an array 🤷.